pkgs/test/README.md

```diff
@@ -173,17 +173,31 @@ description (including any group descriptions) match that regular expression
 will be run. You can also use the `-N` flag to run tests whose names contain a
 plain-text string.
 
-Alternatively, you can filter tests by name only for a specific file/directory
-by specifying a query on a path: `dart test "path/to/test.dart?name=test name"`.
-That ensures that the name filters applies to one path but not others.
-
 By default, tests are run in the Dart VM, but you can run them in the browser as
 well by passing `dart test -p chrome path/to/test.dart`. `test` will take
 care of starting the browser and loading the tests, and all the results will be
 reported on the command line just like for VM tests. In fact, you can even run
 tests on both platforms with a single command: `dart test -p "chrome,vm"
 path/to/test.dart`.
 
+### Test Path Queries
+
+Some query parameters are supported on test paths, which allow you to filter the
+tests that will run within just those paths.
+
+- **name**: Works the same as `--name` (simple contains check).
+  - This is the only option that supports more than one entry.
+- **full-name**: Requires an match for the name of the test.
+- **line**: Matches any test that originates from this line in the test suite.
+  - Not supported for browser tests.
+- **col**: Matches any test that originates from this column in the test suite.
```

Comment:
I'm confused who would use this? Is this a specific column? Like the position of the first `t` in `test(`?

Reply:
I still don't understand the precise position this matches.

_Is_ it exactly the first `t` in `test`? Is it 0 based? Does this match the same frame as the line number, or can the line match one frame and the column match a different frame?

I'll read through the test cases tomorrow to find out more, but I do think we should expand the user facing docs.